src/plugins/home/public/application/components/guided_onboarding/use_case_card.tsx

```diff
@@ -136,40 +134,11 @@ export const UseCaseCard = ({ useCase }: UseCaseProps) => {
       <p>{constants[useCase].i18nTexts.description}</p>
     </EuiText>
   );
-
-  const { euiTheme } = useEuiTheme();
-  const isSmallerBreakpoint = useIsWithinBreakpoints(['xs', 's']);
-  const isMediumBreakpoint = useIsWithinBreakpoints(['m']);
-  const cardCss = useMemo(() => {
-    return {
-      backgroundColor:
-        useCase === 'search'
-          ? euiTheme.colors.warning
-          : useCase === 'security'
-          ? euiTheme.colors.accent
-          : euiTheme.colors.success,
-      // smaller screens: taller cards (250px)
-      // medium screens: lower cards (150px)
-      // larger screens: tall but not too tall cards (200px)
-      minHeight: isSmallerBreakpoint ? 250 : isMediumBreakpoint ? 150 : 200,
-    };
-  }, [euiTheme, isMediumBreakpoint, isSmallerBreakpoint, useCase]);
-
   return (
     <EuiCard
       display="subdued"
       textAlign="left"
-      icon={
-        <EuiAvatar
-          iconSize="xl"
-          iconType={constants[useCase].icon.type}
-          name={constants[useCase].icon.name}
-          color="plain"
-          size="xl"
-          // TODO add useEuiShadow('m') when EUI import is available (https://github.com/elastic/eui/pull/5970)
-        />
-      }
-      image={<div css={cardCss} />}
+      image={<img src={getImageUrl(useCase)} alt={constants[useCase].logo.altText} />}
```

Comment:
This should be using either a plain URL or `<EuiImage/>`.

Reply:
I updated this to use `EuiImage` - a plain URL renders and empty `alt` tag ([code](https://github.com/elastic/eui/blob/main/src/components/card/card.tsx#L242))